<?xml version="1.0"?>
<launch>
    <arg name="robot_env" default="$(optenv ROBOT_ENV !!NO_ROBOT_ENV_SET!!)" />
    <arg name="move_arm_server" default="move_arm_server" />
    <arg name="move_base_server" default="/move_base" />
    <arg name="pose_frame" default="map" />
    <arg name="safe_arm_joint_config" default="folded" />

    <node pkg="mdr_move_base_action" type="move_base_action" name="move_base_server" output="screen">
        <param name="move_arm_server" value="$(arg move_arm_server)" />
        <param name="move_base_server" value="$(arg move_base_server)" />
        <param name="pose_frame" value="$(arg pose_frame)" />
        <param name="safe_arm_joint_config" value="$(arg safe_arm_joint_config)" />
        <param name="pose_description_file" value="$(find mdr_environments)/$(arg robot_env)/navigation_goals.yaml" />
    </node>
</launch>
